packages/wrangler/src/__tests__/queues.test.ts

```diff
@@ -327,6 +332,167 @@ describe("wrangler", () => {
+
+				msw.use(
+					http.put(
+						"*/accounts/:accountId/queues/:queueId",
+						async ({ request }) => {
+							requests.count += 1;
+
+							const body = (await request.json()) as {
+								queue_name: string;
+								settings: {
+									delivery_delay: number;
+									message_retention_period: number;
+								};
+							};
+							expect(body.queue_name).toEqual(queueName);
+							expect(body.settings).toEqual(queueSettings);
+							return HttpResponse.json({
+								success: true,
+								errors: [],
+								messages: [],
+								result: {
+									queue_name: queueName,
+									created_on: "01-01-2001",
+									modified_on: "01-01-2001",
+								},
+							});
+						},
+						{ once: true }
+					)
+				);
+				return requests;
+			}
+			function mockGetQueueRequest(
+				queueName: string,
+				queueSettings: {
+					delivery_delay: number;
+					message_retention_period: number;
+				}
+			) {
+				const requests = { count: 0 };
+				msw.use(
+					http.get(
+						"*/accounts/:accountId/queues?*",
+						async () => {
+							requests.count += 1;
+							return HttpResponse.json({
+								success: true,
+								errors: [],
+								messages: [],
+								result: [
+									{
+										queue_name: queueName,
+										created_on: "",
+										producers: [],
+										consumers: [],
+										producers_total_count: 1,
+										consumers_total_count: 0,
+										modified_on: "",
+										queue_id: "queueId",
+										settings: {
+											delivery_delay: queueSettings.delivery_delay,
+											message_retention_period:
+												queueSettings.message_retention_period,
+										},
+									},
+								],
+							});
+						},
+						{ once: true }
+					)
+				);
+				return requests;
+			}
+
+			it("should show the correct help text", async () => {
+				await runWrangler("queues update --help");
+				expect(std.err).toMatchInlineSnapshot(`""`);
+				expect(std.out).toMatchInlineSnapshot(`
+					"wrangler queues update <name>
+
+					Update a Queue
+
+					POSITIONALS
+					  name  The name of the queue  [string] [required]
+
+					GLOBAL FLAGS
+					  -j, --experimental-json-config  Experimental: support wrangler.json  [boolean]
+					  -c, --config                    Path to .toml configuration file  [string]
+					  -e, --env                       Environment to use for operations and .env files  [string]
+					  -h, --help                      Show help  [boolean]
+					  -v, --version                   Show version number  [boolean]
+
+					OPTIONS
+					      --delivery-delay-secs            How long a published message should be delayed for, in seconds. Must be between 0 and 42300  [number]
+					      --message-retention-period-secs  How long to retain a message in the queue, in seconds. Must be between 60 and 1209600  [number]"
+				`);
+			});
+
+			it("should update a queue with new message retention period and preserve old delivery delay", async () => {
```

Comment:
should `update` include a test case that tests for incorrectly passing the new args? Like [here](https://github.com/cloudflare/workers-sdk/pull/6450/files#diff-22fb18026d024607b9500ed4ec29b341b48af774b169f1bc20a7053d0a420276R350)

Reply:
whoohoo! thank you 🚀